nemo_text_processing/inverse_text_normalization/en/taggers/telephone.py

```diff
@@ -28,12 +29,26 @@
 
 def get_serial_number(cardinal):
     """
-    any alphanumerical character sequence with at least one number with length greater equal to 3
+    any alphanumerical character sequence with at least one number with length greater equal to 3 and
+    excluding any numeric sequence preceded by 'a'
```

Comment:
why excluding 'a'?

Reply:
Updated the comment to read as follows " ... excluding any numeric sequence containing double digits (ties/teens) preceded by 'a'. This avoids cases like "a thirty six" being converted to "a36"  in "a thirty six times increase"

Reply:
> Updated the comment to read as follows '_... excluding any numeric sequence containing double digits (ties/teens) preceded by 'a'. This avoids cases like "a thirty six" being converted to "a36" in "a thirty six times increase"_'

makes sense! thank you.